ansible/roles/kong-api/defaults/main.yml

```diff
@@ -12984,3 +12984,21 @@ kong_apis:
        config.limit_by: credential
      - name: request-size-limiting
        config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: workflowGetUserWFApplicationFieldsV2
+    uris: "{{ workflow_handler_service_prefix }}/v2/getUserWFApplicationFields"
+    upstream_url: "{{ workflow_handler_service_url }}/v1/workflow/v2/getUserWFApplicationFields"
```

Comment:
why version is mentioned twice ?
upstream value should be - 
upstream_url: "{{ workflow_handler_service_url }}/v2/workflow/getUserWFApplicationFields"